redfish-core/lib/update_service.hpp

```diff
@@ -764,6 +763,25 @@ inline void
     }
 }
 
+/**
+ * UpdateServiceActionsOemConcurrentUpdate class supports handle POST method for
+ * concurrent update action.
+ */
+inline void requestRoutesUpdateServiceActionsOemConcurrentUpdate(App& app)
+{
+    BMCWEB_ROUTE(app, "/redfish/v1/UpdateService/Actions/Oem/"
+                      "OemUpdateService.ConcurrentUpdate/")
```

Comment:
Shouldn't the URL be all in one string to align with bmcweb common errors:
[https://github.com/openbmc/bmcweb/blob/master/COMMON_ERRORS.md#10-urls-constructed-in-aggregate](https://github.com/openbmc/bmcweb/blob/master/COMMON_ERRORS.md#10-urls-constructed-in-aggregate)

Reply:
Agreed. And the new clang-format is okay with that 
https://github.com/openbmc/bmcweb/commit/0fda0f12bb9ae0604a083dfae419be38a1418913

Reply:
Thanks. I updated it